core/src/main/scala/sttp/tapir/internal/ShowPathTemplate.scala

```diff
@@ -37,10 +37,10 @@ object ShowPathTemplate {
   ): (Vector[String], Int) =
     inputs.foldLeft((Vector.empty[String], 1)) { case ((acc, index), component) =>
       component match {
-        case p: EndpointInput.PathCapture[_]  => (acc :+ showPathParam(index, p), index + 1)
-        case _: EndpointInput.PathsCapture[_] => (showPathsAs.fold(acc)(acc :+ _), index)
-        case EndpointInput.FixedPath(s, _, _) => (acc :+ UrlencodedData.encodePathSegment(s), index)
-        case _                                => (acc, index)
+        case p: EndpointInput.PathCapture[_] if !p.codec.schema.hidden  => (acc :+ showPathParam(index, p), index + 1)
```

Comment:
ah! good catch, we should hide these inputs here as well. Maybe it would be worth adding a test in `EndpointTest:257`?